lib/email_address.rb

```diff
@@ -40,10 +40,10 @@ module EmailAddress
   #   secret to use in options[:secret]
   class << self
     (%i[valid? error normal redact munge canonical reference base srs] &
-     EmailAddress::Address.public_instance_methods
+     Address.public_instance_methods
     ).each do |proxy_method|
       define_method(proxy_method) do |*args, &block|
-        EmailAddress::Address.new(*args).public_send(proxy_method, &block)
+        Address.new(*args).public_send(proxy_method, &block)
```

Comment:
Have you test this after constant name change mentioned in README? Maybe, `self.class::Address` is better.

Reply:
Yeah that's working fine, same as `def self.` methods. `EmailAddress` is still in its ancestors. I cleared up the code a bit and added some tests.

Reply:
I meant, maybe Ruby internally remembers this reference strongly as `EmailAddress::Address`, and following renaming of outer constant doesn't affect this internal constant. But I glad to hear that it works, thank you.